overview/content-types/option.nix

```diff
@@ -84,11 +84,15 @@ in
           alert-update-script =
             let
               isDrv = self.type == "package";
```

Comment:
Side note: shouldn't the type field be types.optionType? Is that information preserved in optionsDoc? Otherwise we're matching against a magic string essentially. Not urgent or important, just a little paper cut one can prevent by being more rigorous.

Reply:
I agree with your point, but I'm not sure we can use this to get the type definitions:

```
nix-repl> lib.types.optionType.getSubOptions {}
{ }
```

Maybe I haven't looked hard enough, but I've looked for a proper way to do this before and couldn't find any ...